includes/block-patterns/course-list/class-sensei-course-list-block-patterns.php

```diff
@@ -129,6 +150,8 @@ public function register_course_list_block_patterns() {
 
 										<!-- wp:sensei-lms/course-overview /-->
 
+										' . implode( "\n", $course_list_extra_links['course-list'] ) . '
```

Comment:
Should this be the `course-grid` key?

```suggestion
										' . implode( "\n", $course_list_extra_links['course-grid'] ) . '
```

Reply:
😱 

https://github.com/Automattic/sensei/pull/6296/commits/664f95325acf6177853c5cc4611477c75e5d2400

Good catch!